### PR TITLE
[Lifecycle] Update default service bind address/hostname

### PR DESCRIPTION
**Pull request by leecalcote**
- misspelling of "adapter"
- bind to localhost from mesherylocal.layer5.io

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -145,7 +145,7 @@ func serviceAddress() string {
 		return svcAddr
 	}
 
-	return "mesherylocal.layer5.io"
+	return "localhost"
```
Comment (Aisuko) [inline main.go:148]: I checked some of adapters and all of them are using `mesherylocal.layer5.io` as service address. Why we need to change this on `meshery-istio`?

Comment (leecalcote) [reply main.go:148]: Great question, @Aisuko. I was just starting here on this adapter somewhat randomly as the first adapter to receive this change. The need for use of `mesherylocal.layer5.io` as an alternate to `localhost` has dissipated. Ultimately, we should be able to remove this note as well - https://docs.meshery.io/project/contributing/contributing-ui#running-meshery-from-ide

Comment (Aisuko) [reply main.go:148]: @leecalcote Looks good. Is there any test case can make sure this work well?  If sure. I can help do PR for other adapters and Helm charts.

Comment (leecalcote) [reply main.go:148]: @navendu-pottekkat @Revolyssup will you confirm that the e2e test cases (that ran here - https://github.com/meshery/meshery-istio/runs/4659835134?check_suite_focus=true) covers this change?

Comment (leecalcote) [reply main.go:148]: @Aisuko, yes, please. Your help in propagating this change to the rest of the adapters would be excellent.

Comment (Aisuko) [reply main.go:148]: @leecalcote No problem.

Comment (Revolyssup) [reply main.go:148]: @leecalcote  in these tests, adapter is build from dockerfile, and it has service address as 
https://github.com/meshery/meshery-istio/blob/d89889fb6dc8f50f817399f363328fed506ece40/Dockerfile#L30
SO no probs

Comment (leecalcote) [reply main.go:148]: Ok, good deal, @Aisuko. 👍 Approved for merge?

**Comment by leecalcote** [comment]
Oops. My merging of the other PR #342 affected this PR because #342 had the same commit included. I think we're ok, though, considering @Aisuko's last, positive comment.

**Comment by Aisuko** [comment]
No problems.

**Pull request merged**